ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml

```diff
@@ -0,0 +1,32 @@
+base_images:
```

Comment:
@jianzhangbjz suggest to change the file to 
```yaml
build_root:
  image_stream_tag:
    name: "4.14"
    namespace: ocp
    tag: tools
images:
- dockerfile_path: prow/Dockerfile
  from: root
  to: job-python3
resources:
  '*':
    limits:
      memory: 8Gi
    requests:
      cpu: "1"
      memory: 4Gi
tests:
- as: stable-build-test
  cron: 0 * * * *
  steps:
    test:
    - ref: api-job
zz_generated_metadata:
  branch: master
  org: openshift
  repo: release-tests

```

Reply:
Great! Thanks!